app/page.js

```diff
@@ -1062,9 +1063,18 @@ export default function Home() {
           </form>
         </section>
         <section className='py-5 cursor-pointer grid gap-10 md:grid-cols-2 lg:grid-cols-3'>
-          {filteredCards.slice(0, visibleItems).map((card, index) => (
-            <TravelCard key={index} {...card} />
-          ))}
+          {filteredCards.slice(0, visibleItems).map((card, index) => {
+            if (card.imageUrl === "/images/planning.png" && 
+              card.title === "Keeping the dream alive by traveling the world.") {
+              return (
+                <Link href="/post" key={index}>
```

Comment:
El link deberia ser parte de la travel card, no al reves. Ademas, recuerda que un tag `a` no deberia tener `div` en su contenido, eso lo habiamos conversado en anteriores actividades. Es hacer algo asi:
```
<div class="relative">
  <a class="absolute inset-0" aria-label="read the post content"></a>
  El resto del contenido
```

Diras que es para ponerselo solo a una card, pero igual puedes enviar eso por atributos donde defines los cardData
```
{
  imageUrl: "/images/planning.png",
  priceColor: "bg-softYellow",
  fill: "currentColor",
  price: "$49/MO",
  title: "Keeping the dream alive by traveling the world.",
  abstract: "Integrate the latest technologies with an innovative platform...",
  link: '/post'
},
```

Y en el componente
`export default function TravelCard({ imageUrl, priceColor, fill, price, title, abstract, link = null }) {`
Y
`{link && <a class="absolute inset-0" aria-label="read the post content"></a>}`